common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationSettings.java

```diff
@@ -57,7 +57,7 @@
 
     private String mBrokerPackageName = AuthenticationConstants.Broker.COMPANY_PORTAL_APP_PACKAGE_NAME;
 
-    private String mBrokerSignature = AuthenticationConstants.Broker.COMPANY_PORTAL_APP_SIGNATURE;
+    private String mBrokerSignature = AuthenticationConstants.Broker.COMPANY_PORTAL_APP_RELEASE_SIGNATURE;
```

Comment:
>COMPANY_PORTAL_APP_RELEASE_SIGNATURE [](start = 69, length = 36)

Should this be Debug Signature in case of running in Debug mode?

Reply:
I see, we use the same signature everywhere.

---
In reply to: [564909272](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1191#discussion_r564909272) [](ancestors = 564909272)

Reply:
We don't know what Company Portal debug's signature is and we don't even know how to get a handle on a debug build of CP. In other words, we never expect to be playing with a debug CP. If in the future, we do get access to debug builds of CP then we can find out its signature and include it here.